templates/index.html.go

```diff
@@ -180,8 +180,8 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
     <script src="{{.comboURL}}?app/assets/javascripts/version.js"></script>
     <script src="{{.comboURL}}?app/assets/javascripts/yui/yui/yui.js"></script>
     {{else}}
-    <script src="{{.comboURL}}?app/assets/javascripts/version.js"></script>
-    <script src="{{.comboURL}}?app/assets/javascripts/yui/yui/yui-min.js"></script>
+    <script src="{{.staticURL}}/static/build/version.json"></script>
```

Comment:
Both of these html templates need considerable amount of cleanup - this will be done in a follow-up